modern-font-stacks-for-wp.php

```diff
@@ -20,27 +20,21 @@
  *
  * @since 1.0
  * 
- * @param object $theme_json The original theme JSON data.
- * @return object The modified theme JSON data.
+ * @param WP_Theme_JSON_Data $theme_json The original theme JSON data.
+ * @return WP_Theme_JSON_Data The modified theme JSON data.
```

Comment:
`WP_Theme_JSON_Data` is guaranteed by the filter.
https://developer.wordpress.org/reference/hooks/wp_theme_json_data_theme/